fcrepo-indexing-solr/pom.xml

```diff
@@ -33,6 +33,11 @@
       <groupId>org.apache.camel</groupId>
       <artifactId>camel-http4</artifactId>
     </dependency>
+
+    <dependency>
+      <groupId>org.apache.camel</groupId>
```

Comment:
Nope, that should be removed. That's from when I started implementing the reindexing piece inside the indexing-solr application. I have a number of changes that are part of a forthcoming commit.